README.md

```diff
@@ -131,6 +155,12 @@ Retrieve the status of shuffle
     - `NO_SHUFFLE` will play the tracks in the order they were added
     - We created constants named `GMusic.Playback.ALL_SHUFFLE` or `GMusic.Playback.NO_SHUFFLE`
 
+#### `playback.setShuffle(mode)`
+Set the shuffle mode
+
+- mode `String` - Required mode to change shuffle to
```

Comment:
Required is implicit and we can't define something with it in its own definition (i.e. `mode` in `mode`). How about:

```
- mode `String` - Value to change shuffle to 
```